### PR TITLE
Update Modify Delivery API documentation

### PR DESCRIPTION
**Pull request by arewg**
Changes to Modify Delivery API

- Added text that we support Sweden and Denmark
- Expanded on list of events that should not be on a shipment
- Added list of VAScodes that should not be on a shipment
- Expanded table of service codes to include the SE/DK ones + one new NO service code
- Added list of limitations to the API (Routes for 03XX service codes, change address only being available on shipments going to norway)
- Added table showing Stop delivery valid routes

### CODE REVIEW

**Review by eideeivindsen — APPROVED** [review]
LGTM! I trust you know your domain;)

File: content/api/modify-delivery/index.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ menu:
...
-  their way to the recipient. Shipments can either be stopped and returned to
-  the sender, or they can be rerouted to a new delivery address. It is also
-  possible to change the cash on delivery amount. At the moment, Parcel Norway
-  domestic services (with the exception of Pakke i postkassen and return
-  services) are supported.
+  The Modify Delivery API can be used to modify shipments that are on
+  their way to the recipient. Shipments can be stopped and returned to
+  the sender, they can be rerouted to a new delivery address or it is
+  possible to change the cash on delivery amount. We support
+  domestic and cross border shipments in Norway, Sweden and Denmark, but
```
Comment (stianste) [inline content/api/modify-delivery/index.md:18]: I would split the sentence here to keep them as short as possible. 

```suggestion
  domestic and cross border shipments in Norway, Sweden and Denmark. Do note however, that there are ...
```

File: content/api/modify-delivery/index.md [hunk 2]
```diff
@@ -11,12 +11,13 @@ menu:
...
+  please note that there are limitations to our services in Sweden and Denmark.
+  Detailed overview for our services can be found below.
```
Comment (stianste) [inline content/api/modify-delivery/index.md:20]:
```suggestion
  Detailed overview of our services can be found below.
```

File: content/api/modify-delivery/index.md [hunk 3]
```diff
@@ -30,14 +31,27 @@ information:
...
         - Deviation
         - Returned
         - Delivered
         - Lost
         - Partly delivered
         - Stop shipment
+        - In transit (to final destination)
+        - Home Delivery ordered
+
+      Any of the following value added services should not already be present on the shipment:
```
Comment (stianste) [inline content/api/modify-delivery/index.md:47]:
```suggestion
      Any of the following value added services (VAS) should not already be present on the shipment:
```

Comment (stianste) [inline content/api/modify-delivery/index.md:47]:
```suggestion
      Any of the following value added services should **not** already be present on the shipment:
```

**Review by stianste — APPROVED** [review]
Looks good as far as I can see 🙌

**Pull request merged**